seed-job/buildtemplate.yaml

```diff
@@ -36,7 +36,9 @@ objects:
                     alwaysPullImage: true,
                     workingDir: '/tmp',
                     privileged: true,
-                    args: '${computer.jnlpmac} ${computer.name}'
+                    args: '${computer.jnlpmac} ${computer.name}',
+                    resourceRequestCpu: '2',
+                    resourceRequestMemory: '5Gi'
```

Comment:
I think the requirement on memory for seed-job is on higher side ?

Also - how about using these values from parameters ?